src/modules/iasql_functions/rpcs/iasql_get_sql_since.ts

```diff
@@ -96,7 +98,7 @@ async function getChangeLogs(limitDate: string, orm: TypeormWrapper): Promise<Ia
 }
 
 /** @internal */
-async function getInstalledModules(orm: TypeormWrapper): Promise<ModuleInterface[]> {
+export async function getInstalledModules(orm: TypeormWrapper): Promise<ModuleInterface[]> {
```

Comment:
So we tend to have shared logic between these RPC functions in the `src/modules/iasql_functions/iasql.ts` file. Perhaps this should be moved there and imported into both RPCs?